### ISSUE TITLE
apt-vim continually looking for python in /usr/bin/env

### ISSUE THREAD

**Issue by caleywoods** [egalpin/apt-vim#58]
I am working from a Samsung Chromebook 3 running what is similar to an ubuntu chroot via crouton. I have python 3.5 located at `/usr/bin/python3.5` and it's aliased in my `.bashrc` with `alias python='/usr/bin/python3.5` and executing `python --version` from the bash shell will return back the python version.

When trying to install NERDTree with `apt-vim`, here is the output:
```
(xenial)caley@localhost:~$ apt-vim install -y https://github.com/scrooloose/nerdtree.git
/usr/bin/env: 'python': No such file or directory
```

If I prefix the above `apt-vim install` command with python then everything works fine. Should I modify `.vimpkg/bin/apt-vim` to point to `/usr/bin/python3.5` or is there a better solution for this?

**Comment by egalpin** [comment]
The better solution is probably for me to make `apt-vim` installable via `pip`, [homebrew](https://brew.sh/), and various linux package managers (yum, apt-get, etc) such that this is handled correctly.

Your initial thought to modify the `apt-vim` python script to point to `/usr/bin/python3.5` would rectify this on your system :-) I can also probably modify the install script to try to handle this more gracefully (i.e. find the system python via `which python` and add that as the shebang at the top of `apt-vim`)

**Comment by caleywoods** [comment]
Yeah, the change at the top of the file took care of it for me locally. `apt-get` should also work at least in my case, that seems to be how packages are installed inside the chroot, some kind of mock ubuntu CLI only type setup since I didn't opt for a window manager.
